docker/compose.yaml

```diff
@@ -752,6 +753,10 @@ services:
       internal:
         ipv4_address: $IPV4_IP_GRAFANA_INTERNAL
 
+    profiles:
+      - alertmanager
```

Comment:
I don't think `alertmanager` needs to be specified here (and the others). Thought doesn't make sense to enable `alertmanager` profile without the `monitoring` profile also enabled. It might be better to keep them separated.